tsconfig.json

```diff
@@ -4,5 +4,6 @@
         "esModuleInterop": true,
         "moduleResolution": "node",
         "resolveJsonModule": true,
+        "strictPropertyInitialization": false,
```

Comment:
I wouldn't set this as false. Being explicit in the initialization forces to thing more on what you are doing.